docs/sessions.md

```diff
@@ -76,44 +88,46 @@ Cookie Serialization<br>_(of particular interest to .NET Core < netstandard2.0)_
 Suave uses the .NET Framework type `BinaryFormatter` to serialize the `Map<string, obj>` containing the session state; this is the default. However, the `BinaryFormatter` was removed in the .NET Core API, and the `DataContractJsonSerializer` does not recognize the `Map<string, obj>` type. One option is to utilize JSON.NET to serialize this object. To use that, ensure you've added the `Newtonsoft.Json` NuGet package to your project, then put the following code somewhere before the `suaveCfg` definition in the example above.
 
 {% highlight fsharp %}
-/// alias
-let utf8 = System.Text.Encoding.UTF8
-
+// NOTE: UTF8 is in the auto-opened module YoLo.
```

Comment:
I prefer the previous version, as YoLo is internal by default https://github.com/SuaveIO/suave/blob/master/Rakefile#L24